src/Arcus.Testing.Tests.Integration/Integration/DataFactory/Fixture/TemporaryDataFactoryDataFlow.cs

```diff
@@ -302,6 +300,12 @@ private async Task AddFlowletsAsync(TempDataFlowOptions dataFlowOptions)
             }
         }
 
+        private static DataFactoryDataFlowResource GetFlowlet(string subscriptionId, string resourceGroupName, string dataFactoryName, ArmClient _arm, string flowletName)
+        {
+            ResourceIdentifier flowletResourceId = DataFactoryDataFlowResource.CreateResourceIdentifier(subscriptionId, resourceGroupName, dataFactoryName, flowletName);
+            return _arm.GetDataFactoryDataFlowResource(flowletResourceId);
+        }
```

Comment:
```suggestion
        private static DataFactoryDataFlowResource GetFlowlet(string subscriptionId, string resourceGroupName, string dataFactoryName, ArmClient arm, string flowletName)
        {
            ResourceIdentifier flowletResourceId = DataFactoryDataFlowResource.CreateResourceIdentifier(subscriptionId, resourceGroupName, dataFactoryName, flowletName);
            return arm.GetDataFactoryDataFlowResource(flowletResourceId);
        }
```
**requires change:** parameter names usually do not have `_` prefix, that's mostly reserved from fields.